scripts/o3de/tests/test_export_project.py

```diff
@@ -544,6 +546,14 @@ def test_setup_launcher_layout_directory(tmp_path, build_config, asset_platform,
         result_archive_file = tmp_path / f"output.{test_archive_output_format}"
         assert result_archive_file.is_file(), f"Missing <output>.{test_archive_output_format}"
 
+    setregpatch_file = test_output_path / 'Registry/IgnoreAssetProcessor.profile.setregpatch'
```

Comment:
Changes need to be rebased, as my PR has this snippet too:
https://github.com/o3de/o3de/pull/17238/files#diff-82f65b6ed50b7919bb5d55e60a11cef546cb0dc86eddfaed47c540aae671d667R548